### PR TITLE
Add way to refresh Dr. CI results from HUD

### PR DESCRIPTION
**Pull request by ZainRizvi**
Adds a `@pytorchbot drci` command which triggers a manual update to the Dr. CI comment.

This will help mitigate times when Dr. CI is stuck displaying stale information

Fixes https://github.com/pytorch/pytorch/issues/88231

### CODE REVIEW

File: torchci/lib/bot/pytorchBotHandler.ts [hunk 1]
```diff
@@ -273,6 +274,18 @@ The explanation needs to be clear on why this is needed. Here are some good exam
     }
   }
 
+
+  async handleDrCI(labels: string[]) {
```
Comment (clee2000) [inline torchci/lib/bot/pytorchBotHandler.ts:278]: why do you need the labels as input?

File: torchci/lib/drciUtils.ts [hunk 2]
```diff
@@ -107,3 +108,58 @@ ${sev_list}\n
 `;
   }
 }
+
+// The context here is the context from probot.
+// Today we only use probot for upserts, but this could later be split into logger
+export async function upsertDrCiComment(owner: string, repo: string, prNum: number, context: any, prUrl: string) {
+  // Dr.CI only supports pytorch/pytorch at the moment
+  context.log("It's me!")
```
Comment (clee2000) [inline torchci/lib/drciUtils.ts:116]: probably dont want this?

**Review by clee2000 — APPROVED** [review]
lgtm except for 2 small things

**Review by huydhn — COMMENTED** [review]
LGTM! Only a minor issue with the failed test.  Let's see how people use this. IMO, i feel faster to go to HUD directly than typing a command to see Dr.CI page refresh :P

File: torchci/lib/drciUtils.ts [hunk 3]
```diff
@@ -107,3 +108,58 @@ ${sev_list}\n
...
+  if (owner != OWNER || repo != REPO) {
```
Comment (malfet) [inline torchci/lib/drciUtils.ts:117]: We have `isPyTorchPyTorch(owner, repo)` function in `utils.ts`

**Review by malfet — APPROVED** [review]

File: torchci/lib/drciUtils.ts (hunk 3 above)
Comment (ZainRizvi) [reply torchci/lib/drciUtils.ts]: Nice, I'll switch to that

**Pull request merged**